geocity/apps/forms/admin.py

```diff
@@ -97,6 +97,14 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
+    # TODO: remove this once M2M relation is removed in model
+    def clean_administrative_entities(self):
+        if len(self.cleaned_data["administrative_entities"]) > 1:
```

Comment:
put `self.cleaned_data["administrative_entities"]` in a variable, to don't retrieve the same info twice